action.yml

```diff
@@ -26,10 +32,14 @@ runs:
         # Run the Julia command
         "${julia_cmd[@]}"
       shell: bash
-
     - run: |
         # The Julia command that will be executed
-        julia_cmd=( julia --color=yes --project=docs/ docs/make.jl )
+        julia_cmd=( julia --color=yes --code-coverage --project=docs/ -e '
```

Comment:
oh, I should call out the `--code-coverage` flag is a separate addition here. I can split that into a separate PR if desired. The reason I think it's really useful is that it means you can get coverage from doctests just running the process-coverage action after.

Reply:
A separate PR or two separate commits would be nice so that we can link to them in the release notes.

Reply:
Ok, I took it out here so this one can be squash-merged and then I’ll make a separate PR to add it